src/mock.jl

```diff
@@ -26,6 +27,7 @@ macro mock(expr)
                 $target($args_var...; $(kwargs...))
             end
         else
+            @debug "Mocking not activated, @mock macro expanding to the original target" $target
```

Comment:
This breaks the promise that the `@mock` macro only adds overhead when activated. A better option for this kind of option would be to provide a warning when a patch is applied but Mocking has not been activated.